### ISSUE TITLE
Support for paragraph indentation

### ISSUE THREAD

**Issue by erenard** [privateOmega/html-to-docx#106]
Proposed solution:
form the given HTML:
```html
<p style="margin-left:40px;"></p>
```
- use `pixelToTWIP` to convert units
- generate the following indentation object:
```js
modifiedAttributes.indentation = { left: pixelToTWIP(40) };
```
Of course right indentation works the same way.

**Comment by privateOmega** [comment]
Merged #107

**Comment by erenard** [comment]
Hello @privateOmega, I would like to use the next version of `html-to-docx` as a dependency in another project.
I need the «html margins to indentation» feature for that project, can you make a new release on npm please ?
